float32 confidence
int8 class_id
float32 angle_left
float32 angle_right
float32 angle_top
float32 angle_bottom

